docs/UsersGuide/source/ConfigNewPlatform.rst

```diff
@@ -28,310 +28,174 @@ The first step to installing on a new machine is to install :term:`NCEPLIBS` (ht
-
-.. _WallClockTimes:
-
-.. table::  Example wallclock times for each workflow task.
-
-
-   +--------------------+----------------------------+------------+-----------+
-   | **UFS Component**  | **Script Name**            | **Num.**   | **Wall**  |
-   |                    |                            | **Cores**  | **time**  |
-   +====================+============================+============+===========+
-   | UFS_UTILS          | ./run_get_ics.sh           | n/a        | 3 s       |
-   +--------------------+----------------------------+------------+-----------+
-   | UFS_UTILS          | ./run_get_lbcs.sh          | n/a        | 3 s       |
-   +--------------------+----------------------------+------------+-----------+
-   | UFS_UTILS          | ./run_make_grid.sh         | n/a        | 9 s       |
-   +--------------------+----------------------------+------------+-----------+
-   | UFS_UTILS          | ./run_make_orog.sh         | 4          | 1 m       |
-   +--------------------+----------------------------+------------+-----------+
-   | UFS_UTILS          | ./run_make_sfc_climo.sh    | 4          | 27 m      |
-   +--------------------+----------------------------+------------+-----------+
-   | UFS_UTILS          | ./run_make_ics.sh          | 4          | 5 m       |
-   +--------------------+----------------------------+------------+-----------+
-   | UFS_UTILS          | ./run_make_lbcs.sh         | 4          | 5 m       |
-   +--------------------+----------------------------+------------+-----------+
-   | ufs-weather-model  | ./run_fcst.sh              | 6          | 1h 40 m   |
-   +--------------------+----------------------------+------------+-----------+
-   | UPP                | ./run_post.sh              | 1          | 7 m       |
-   +--------------------+----------------------------+------------+-----------+
-
-Running on a New Platform with Rocoto Workflow Manager
-======================================================
-All official HPC platforms for the UFS SRW App release make use of the Rocoto workflow management software for running experiments. If you would like to use the Rocoto workflow manager on a new machine, you will have to make modifications to the scripts in the ``regional_workflow`` repository. The easiest way to do this is to search the files in the ``regional_workflow/scripts`` and ``regional_workflow/ush`` directories for an existing platform name (e.g. ``CHEYENNE``) and add a stanza for your own unique machine (e.g. ``MYMACHINE``). As an example, here is a segment of code from ``regional_workflow/ush/setup.sh``, where the highlighted text is an example of the kind of change you will need to make:
+..
+   COMMENT: Suggested sub-bullets for Meteorology/NWP? 
```

Comment:
Having a general understanding of at least basic NWP and the meteorology at the scales that are being predicted should be considered a prerequisite.